gulpfile.js

```diff
@@ -94,10 +124,42 @@ gulp.task('build', function(cb) {
 			gulp.src(paths.src)
 				.pipe(gulp.dest(path.join(getBuildDir(browser), paths.dest)));
 		});
+
+		if (config[browser].manifest) {
+			del(path.join(getBuildDir(browser), config[browser].manifest));
```

Comment:
Gulp dest seemed to be throwing "could not write, file exists" errors.. Oh yeah, I suppose I should wait to add new files into old delete is complete.